calendars/numpy.yaml

```diff
@@ -46,8 +46,8 @@ events:
     description: |
       The fortnightly NumPy call for newcomers.
 
-    begin: 2022-01-17 16:00:00 +00:00
-    end: 2022-01-17 17:00:00 +00:00
+    begin: 2022-01-10 16:00:00 +00:00
+    end: 2022-01-10 17:00:00 +00:00
```

Comment:
01-10-22 is still a Monday, should this be 01-13-22 (i.e. a Thursday) instead?

Reply:
Or maybe it was supposed to be 2022-02-10, which was a Thursday :joy:

Reply:
I obviously messed up the dates - sorry 😅